ModuleChangelog.md

```diff
@@ -1,3 +1,12 @@
+## 2.5.0
+
+Release Date: July 19, 2023
+
+#### RELEASE NOTES
+
+```
+* Reverse Migration
```

Comment:
This needs to be expanded on, even a brief description of what this allows you to do and what's required (i.e. 'Added an option to the GUI tool which allows users to undo a user migration (converts a migrated local account back to itss original AD account if possible).